main/actions/src/main/scala/sbt/Compiler.scala

```diff
@@ -80,6 +95,13 @@ object Compiler {
       val scalac = scalaCompiler(instance, cpOptions)
       new Compilers(scalac, javac)
     }
+  @deprecated("Deprecated in favor of new sbt.compiler.javac package.", "0.13.8")
+  def compilers(instance: ScalaInstance, cpOptions: ClasspathOptions, javac: JavaTool, ivyConfiguration: IvyConfiguration)(implicit app: AppConfiguration, log: Logger): Compilers =
```

Comment:
Yeah, maybe just inline the method in line 72 and let's not add a new, already deprecated, method.